docs/sources/datasources/elasticsearch/configure-elasticsearch-data-source.md

```diff
@@ -0,0 +1,153 @@
+Administrators can also [configure the data source via YAML]({{< relref "#provision-the-data-source" >}}) with Grafana's provisioning system.
+
+## Configure the data source
+
+To add the Elasticsearch data source, complete the following steps:
+
+1. Click **Connections** in the left-side menu.
+1. Under **Connections**, click **Add new connection**.
+1. Enter `Elasticsearch` in the search bar.
+1. Select **Elasticsearch data source**.
+1. Click **Create a Elasticsearch data source** in the upper right.
+
+You will be taken to the **Settings** tab where you will set up your Elasticsearch configuration.
+
+## Configuration options
+
+The following is a list of configuration options for Elasticsearch.
+
+The first option to configure is the name of your connection:
+
+- **Name** - The data source name. This is how you refer to the data source in panels and queries. Examples: elastic-1, elasticsearch_metrics.
+
+- **Default** - Toggle to select as the default name in dashboard panels. When you go to a dashboard panel this will be the default selected data source.
+
+### HTTP section
+
+- **URL** - The URL of your Elasticsearch server. If your Elasticsearch server is local, use `<http://localhost:9200>`. If it is on a server within a network, this is the URL with port where you are running Elasticsearch. Example: `<http://elasticsearch.example.orgname:9200>`.
+
+- **Allowed cookies** - Specify cookies by name that should be forwarded to the data source. The Grafana proxy deletes all forwarded cookies by default.
+
+- **Timeout** - The HTTP request timeout. This must be in seconds. There is no default, so this setting is up to you.
+
+### Auth section
+
+There are several authentication methods you can choose in the Authentication section.
+
+{{% admonition type="note" %}}
+Use TLS (Transport Layer Security) for an additional layer of security when working with Elasticsearch. For information on setting up TLS encryption with Elasticsearch see [Configure TLS](https://www.elastic.co/guide/en/elasticsearch/reference/8.8/configuring-tls.html#configuring-tls). You must add TLS settings to your Elasticsearch configuration file **prior** to setting these options in Grafana.
+{{% /admonition %}}
+
+- **Basic authentication** - The most common authentication method. Use your `data source` user name and `data source` password to connect.
+
+- **With credentials** - Toggle to enable credentials such as cookies or auth headers to be sent with cross-site requests.
+
+- **TLS client authentication** - Toggle to use client authentication. When enabled, add the `Server name`, `Client cert` and `Client key`. The client provides a certificate that is validated by the server to establish the client's trusted identity. The client key encrypts the data between client and server.
+
+- **With CA cert** - Toggle to authenticate with a CA certificate. Follow the instructions of the CA (Certificate Authority) to download the certificate file.
```

Comment:
Should this be "Toggle to authenticate with a Certificate Authority (CA). Follow the instructions of the CA to download..."?

Reply:
change made